lib/lint.js

```diff
@@ -77,6 +77,7 @@ module.exports = ({ args }) => {
       parser: 'babel-eslint',
     },
   })
+  console.log('running eslint cli')
```

Comment:
the log statement is necessary b/c otherwise console output may be misleading. for example, eslint prints "rerun the command with `--fix`" and users may erroneously think that they should run `ace lint --fix`